web/themes/custom/votegov/src/sass/uswds-overrides/usa-nav.scss

```diff
@@ -62,6 +62,17 @@
         }
       }
     }
+
+    >button {
+      [dir="rtl"] & {
+        text-align: right;
+
+        span:after {
+          right: unset;
+          left: 0;
+        }
+      }
+    }
```

Comment:
remove the `[dir="rtl"]`.
Update text align to be `text-align: start;`
Replace left alignment with `inset-inline-end: 0;`